dom.bs

```diff
@@ -1419,6 +1419,103 @@ that gave folks all the wrong ideas. <a>Events</a> do not represent or cause act
 can only be used to influence an ongoing one.
 
 
+<h3 id=controlling-promises>Controlling {{Promise}}s</h3>
```

Comment:
This actually isn't really specific to promises. For example it could be used to cancel a stream:

```js
const readableStream = getStream({ signal });

// ... later ...
signal.abort(); // basically the same as readableStream.cancel()
```